examples/env_configs/single_agent/sac_training.yml

```diff
@@ -1,6 +1,6 @@
 algorithm: sac
-project: "stable_baselines3" 
-total_timesteps: 5e6
+project: "paper-tests" 
```

Comment:
Probably want to change this project name to something less experimental, maybe `paper-experiments` ?

Reply:
done